src/main/java/depromeet/lessonfour/server/term/api/dto/response/TermItemDto.java

```diff
@@ -0,0 +1,3 @@
+package depromeet.lessonfour.server.term.api.dto.response;
+
+public record TermItemDto(String termTitle, String termContent) {}
```

Comment:
요거 저번에도 말씀드렸는데, 불필요한 prefix들은 없애면 좋을 것 같습니다!

```suggestion
public record TermItemDto(String title, String content) {}
```

Reply:
컨텍스트 접두어 붙이는 게 습관이었어서 깜빡했네용,,
엔드포인트가 컨텍스트 충분히 제공하니 간단히 하는게 더 좋을 것 같습니다 코멘트 반영했습니다!